### PR TITLE
UnicornSoft.UnicornHTTPS version 1.2.64.92

### PR DESCRIPTION
**Pull request by alexkoala**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/19940)

### CODE REVIEW

File: manifests/u/UnicornSoft/UnicornHTTPS/1.2.64.92/UnicornSoft.UnicornHTTPS.locale.ko-KR.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: UnicornSoft.UnicornHTTPS
+PackageVersion: 1.2.64.92
+PackageLocale: ko-KR
+Publisher: Unicorn Soft
+PackageName: Unicorn HTTPS
```
Comment (ItzLevvie) [inline manifests/u/UnicornSoft/UnicornHTTPS/1.2.64.92/UnicornSoft.UnicornHTTPS.locale.ko-KR.yaml:8]:
```suggestion
PackageName: 유니콘 HTTPS
```

File: manifests/u/UnicornSoft/UnicornHTTPS/1.2.64.92/UnicornSoft.UnicornHTTPS.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: UnicornSoft.UnicornHTTPS
+PackageVersion: 1.2.64.92
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://api.unicorn-soft.com/unicornhttps/v1/download/windows
```
Comment (ItzLevvie) [inline manifests/u/UnicornSoft/UnicornHTTPS/1.2.64.92/UnicornSoft.UnicornHTTPS.installer.yaml:9]:
```suggestion
  InstallerUrl: https://unicornhttps-production.s3.ap-northeast-2.amazonaws.com/unicornhttps/v1/hello/windows/unicorn-https-setup.exe
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/u/UnicornSoft/UnicornHTTPS/1.2.64.92/UnicornSoft.UnicornHTTPS.locale.ko-KR.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: UnicornSoft.UnicornHTTPS
+PackageVersion: 1.2.64.92
+PackageLocale: ko-KR
+Publisher: Unicorn Soft
+PackageName: Unicorn HTTPS
+License: Copyright(C) 2019 Unicorn Soft
+ShortDescription: Unicorn HTTPS
```
Comment (ItzLevvie) [inline manifests/u/UnicornSoft/UnicornHTTPS/1.2.64.92/UnicornSoft.UnicornHTTPS.locale.ko-KR.yaml:10]:
```
ShortDescription: Unicorn HTTPS
```
Please write a better description.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/u/UnicornSoft/UnicornHTTPS/1.2.64.92/UnicornSoft.UnicornHTTPS.locale.ko-KR.yaml (hunk 1 above)
Comment (alexkoala) [reply manifests/u/UnicornSoft/UnicornHTTPS/1.2.64.92/UnicornSoft.UnicornHTTPS.locale.ko-KR.yaml:8]: @ItzLevvie Can PackageName localized?

Comment (ItzLevvie) [reply manifests/u/UnicornSoft/UnicornHTTPS/1.2.64.92/UnicornSoft.UnicornHTTPS.locale.ko-KR.yaml:10]: Yes, it has to match Control Panel.

**Comment by alexkoala** [comment]
fixed

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**